src/renderer/dx/DxRenderer.cpp

```diff
@@ -959,12 +958,20 @@ void DxEngine::_InvalidOr(RECT rc) noexcept
         origin.y = static_cast<float>(coord.Y * _glyphCell.cy);
 
         // Create the text layout
-        CustomTextLayout layout(_dwriteFactory.Get(),
-                                _dwriteTextAnalyzer.Get(),
-                                _dwriteTextFormat.Get(),
-                                _dwriteFontFace.Get(),
-                                clusters,
-                                _glyphCell.cx);
+        if (_drawingTextLayout == nullptr)
+        {
+            _drawingTextLayout = ::Microsoft::WRL::Details::Make<CustomTextLayout>(_dwriteFactory.Get(),
+                                                    _dwriteTextAnalyzer.Get(),
+                                                    _dwriteTextFormat.Get(),

```

Comment:
Should we be using `Microsoft::WRL::Make` for all the other `ComPtr`s we make in this class too? If so, maybe we should file a follow up work item.